tests/Behat/Gherkin/Cucumber/CompatibilityTest.php

```diff
@@ -117,73 +106,77 @@ public function testBadFeaturesDoNotParse(SplFileInfo $file)
         $this->parser->parse(file_get_contents($gherkinFile), $gherkinFile);
     }
 
-    public static function goodCucumberFeatures()
+    /**
+     * @return iterable<string, array{file: SplFileInfo}>
+     */
+    public static function goodCucumberFeatures(): iterable
     {
-        return self::getCucumberFeatures('/good');
+        yield from self::getCucumberFeatures('/good');
     }
 
-    public static function badCucumberFeatures()
+    /**
+     * @return iterable<string, array{file: SplFileInfo}>
+     */
+    public static function badCucumberFeatures(): iterable
     {
-        return self::getCucumberFeatures('/bad');
+        yield from self::getCucumberFeatures('/bad');
```

Comment:
why using `yield from` instead of returning the original iterable directly ?

Reply:
Hmm, in this context it's not so helpful. I'll change it back.